executor/batch_point_get.go

```diff
@@ -141,6 +141,10 @@ func (e *BatchPointGetExec) Next(ctx context.Context, req *chunk.Chunk) error {
 		}
 	}
 
+	if e.lock {
```

Comment:
We can remove it now.

Reply:
Why? If the table is not a partition table, initialize function will not add the table delta from `tblInfo.ID`

Reply:
Why not put this logic into initialize() too?

Reply:
There has a lot of return point in `initialize` function, so move it after `initialize` function return should be good.